app/run_clarity.ps1

```diff
@@ -47,7 +47,7 @@ if (!$PythonInstallPath) {
             Exit
         } else {
             LogWrite "Launching Python 3.11 installer and installing Python for you. Please wait."
-            .\python-3.11.3.exe /passive InstallAllUsers=1 PrependPath=1 Include_doc=0 Include_tcltk=0 Include_test=0 Shortcuts=0 SimpleInstallDescription="Installing necessary components for dqxclarity." | Out-Null
+            .\python-3.11.3.exe /passive InstallAllUsers=1 PrependPath=1 Include_doc=0 Include_tcltk=1 Include_test=0 Shortcuts=0 SimpleInstallDescription="Installing necessary components for dqxclarity." | Out-Null

```

Comment:
This is part of the change. We'll also have to handle existing installations that don't have this toggled. This is just a note that we have to implement that first or this PR won't work for existing installations.